src/app/components/ShareAreaLinkButton.tsx

```diff
@@ -30,7 +30,7 @@ export const ShareAreaLinkButton: React.FC<{ uuid: string, areaName: string }> =
           setClicked(true)
         }}
       >
-        <LinkSimple size={20} /><span className='hidden md:inline'>Share</span>
+        <ShareNetwork  size={20} /><span className='hidden md:inline'>Share</span>
```

Comment:
As discussed in #1079, I suggest leaving the icon as is and changing the text from "Share" to "Copy Link"